Directory.Build.props

```diff
@@ -4,9 +4,17 @@
     <Authors>Microsoft.Toolkit</Authors>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <NoPackageAnalysis>true</NoPackageAnalysis>
-    <PackageIconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</PackageIconUrl>
-    <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/license.md</PackageLicenseUrl>
+  <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
+<package>
+  <metadata>
+    <icon>images\nuget.png</icon>
+    <license type="file">License.md</license>
```

Comment:
I don't think we put these in a package/metadata tags, it should just be another tag alongside the `PackageProjectUrl` named `<PackageLicenseFile>License.md</PackageLicenseFile>` and then we use `PackageIcon` for the icon.

According to [the doc](https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#pack-scenarios) we should keep the `PackageIconUrl` as well for now [and have both](https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packageicon).